src/lib/y2storage/storage_manager.rb

```diff
@@ -187,6 +191,7 @@ def staging_changed?
     #
     # Beware: this method can cause data loss
     def commit
```

Comment:
It would be nice to explain the implications of activating `Yast2::FsSnapshot.configure_on_install`, or simply to refer to some documentation (if exists).

Reply:
Well, the method is documented via yardoc in yast-yast2. But I can add a sentence, of course.